inventory/group_vars/all.yml

```diff
@@ -12,8 +12,11 @@ user: ofn-admin
 # to connect using SSH instead of just passwords.
 ssh_pubkey: "{{ lookup('file', '~/.ssh/{{ ssh_pubkey_name }}') }}"
 
-git_repo: https://github.com/openfoodfoundation/openfoodnetwork.git
-git_version: master
+# You can paste the repo/branch straight from a github PR here, like: "Contributor22:new_feature"
```

Comment:
While this is really cool, we lose some flexibility here. If somebody named their repo differently or hosted it somewhere else (because Github is now Microsoft and evil), they can't do it. Can we make both possible?

I also think it would be better to stage the pull request merged into master to test the final result. Github provides that commit: `git fetch origin pull/2479/merge`

I think instead of `git_version: master`, you could define `git_version: pull/2479/merge`. That gives you a copy and paste solution already. I haven't tested that yet. I would actually like to define that on the command line so that we don't have to change the config to stage.

Reply:
If someone had a pull request that wasn't  from a fork of openfoodnetwork or used a non-github repo (has that ever happened in the history of ofn?) we can completely ignore the github_key and change git_repo and git_version, which is what we would do anyway.

Reply:
I think the `--extra-vars` argument would allow us doing both. So if you don't go with the `repo:branch` you can do `--extra-vars "git_repo=foo git_version=bar" and that would have precedenve over the ones in this file. You can check it out at [Passing Variables On The Command Line](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#passing-variables-on-the-command-line).

Reply:
Ah, okay. That's cool. I still think it would be good to have a process to deploy the merge commit of a pull request, but this a good step in the right direction already.